.tekton/stonesoupe-e2e-test.yaml

```diff
@@ -0,0 +1,249 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: stonesoupe-e2e-test
```

Comment:
stonesoup?

Reply:
> stonesoup?

yes, this CI will launch part of [e2e-tests](https://github.com/redhat-appstudio/e2e-tests) tests

Reply:
No I meant there is typo error with stonesoup

Reply:
Thank you for the check. Fixed